iBudget/income_history/views.py

```diff
@@ -65,6 +65,11 @@ def track(request):
                        .name})
         incomes_funds_ids[counter[0]].update(
             {'fund': FundCategories.objects.get(id=incomes_funds_ids[counter[0]]['fund']).name})
+    set_for_chart = set()
+    for num in range(len(incomes_funds_ids)):
+        set_for_chart.add(incomes_funds_ids[num]['fund'])
+    incomes_funds_ids.append(list(set_for_chart))
+    print(incomes_funds_ids)
```

Comment:
It is better to use loggers instead of just printing stuff.

Reply:
this was a temporary print for testing which I forgot to delete.

I will erase it for sure.